src/ring/middleware/json.clj

```diff
@@ -100,12 +102,19 @@
        (handler request respond raise)
        (respond malformed-response)))))
 
+(defn- json-stream
+  "Adapts the output stream from Cheshire to the input stream required by Ring."
```

Comment:
Can you remove this docstring? It's not a public API so it doesn't need one.

Reply:
i’ll change the method name then to better express intent.